mesos-docker/run/mit-application.conf.template

```diff
@@ -0,0 +1,18 @@
+#Provide full path to the mesos native library. For ubuntu its filename is libmesos.so, for mac its libmesos.dylib
```

Comment:
can we put the template file in `test-runner`? Having 2 copies of the same file just means they will become out of sync.
And the one in `test-runner` has to be edited anyway, so having the placeholders in it doesn't really hurt.